.travis.yml

```diff
@@ -85,7 +85,7 @@ jobs:
 
     - stage: deploy
       name: Push Docker image
-      if: branch = master AND tag IS present
+      if: type != pull_request AND tag IS present
```

Comment:
I'm seeing similar config [here](https://github.com/operator-framework/operator-sdk/blob/master/.travis.yml#L50) but I don't understand the reason. Does this mean we can push images to the docker hub even in other branches and why do we want to do that? Thanks!

Reply:
This means that we only want to push images when we are not in a PR and a tag is present, i.e., when we create a GitHub release. The previous condition was never met because when a tag is present, the branch is never `master`, instead, the branch is equal to the tag name. For example, [this build](https://travis-ci.org/pravega/pravega-operator/builds/509965043) should have triggered a deploy, but it didn't.

Reply:
Got it, Thanks.